watcher/watcher.go

```diff
@@ -542,17 +548,12 @@ func ZcashNetParams(network multichain.Network) *zcash.Params {
 }
 
 func NetParams(network multichain.Network, chain multichain.Chain) *chaincfg.Params {
-	switch chain {
-	case multichain.Bitcoin, multichain.BitcoinCash:
-		switch network {
-		case multichain.NetworkMainnet:
-			return &chaincfg.MainNetParams
-		case multichain.NetworkDevnet, multichain.NetworkTestnet:
-			return &chaincfg.TestNet3Params
-		default:
-			return &chaincfg.RegressionNetParams
-		}
+	switch network {
+	case multichain.NetworkMainnet:
+		return &chaincfg.MainNetParams
+	case multichain.NetworkDevnet, multichain.NetworkTestnet:
+		return &chaincfg.TestNet3Params
 	default:
-		panic(fmt.Errorf("cannot get network params: unknown chain %v", chain))
+		return &chaincfg.RegressionNetParams
 	}
 }
```

Comment:
Just to be sure this `NetParams` is used only for the appropriate chains (and having a fail-early approach):

```go
func NetParams(network multichain.Network, chain multichain.Chain) *chaincfg.Params {
	switch chain {
	case multichain.Bitcoin, multichain.BitcoinCash, multichain.DigiByte, multichain.Dogecoin:
		switch network {
		case multichain.NetworkMainnet:
			return &chaincfg.MainNetParams
		case multichain.NetworkDevnet, multichain.NetworkTestnet:
			return &chaincfg.TestNet3Params
		default:
			return &chaincfg.RegressionNetParams
		}
	default:
		panic(fmt.Errorf("cannot get network params: unknown chain %v", chain))
	}
}
```